lib/rexml/source.rb

```diff
@@ -2,6 +2,7 @@
 # frozen_string_literal: false
 
 require "strscan"
+require "stringio"
```

Comment:
Could you sort this list in alphabetical order?

```suggestion
require "stringio"
require "strscan"
```